### PR TITLE
add trusted contributor support

### PR DESCRIPTION
**Pull request by fries1234**
this pr adds trusted contributor support to the bot and it improves the code a bit and it improves the formatting.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -1,50 +1,68 @@
...
                             total_votes_against += 1
                 print(f"Total votes for resolution: {str(total_votes_for)}")
                 print(f"Total votes against resolution: {str(total_votes_against)}")
-                if (total_votes_for/len(core_team_logins)) > 0.5:
+                if (total_votes_for / len(crystal_logins)) > 0.5:
                     print("This RFC should be passed. Adding comment.")
                     if not issue.locked:
-                        issue.create_comment("This resolution has enough votes from Core Team mebers to be considered passed. Please close the issue once appropriate actions have been taken. - Beep Boop (I'm a bot, and this action was performed automagically)")
+                        issue.create_comment(
+                            "This resolution has enough votes from Core Team members to be considered passed. "
```
Comment (SomethingGeneric) [inline bot.py]: Nice work, can you just change this to "enough votes" :sweat_smile: then we'll send it!

**Review by SomethingGeneric — CHANGES_REQUESTED** [review]

File: bot.py (hunk 1 above)
Comment (fries1234) [reply bot.py]: it seems like its already at "enough votes"

Comment (SomethingGeneric) [reply bot.py]: mb head empty i meant remove "from Core Team members" lol

Comment (fries1234) [reply bot.py]: ok did that

**Comment by SomethingGeneric** [comment]
Danke!

**Review by SomethingGeneric — APPROVED** [review]

**Pull request merged**